src/platform/packages/private/kbn-ui-shared-deps-src/webpack.config.js

```diff
@@ -20,8 +20,10 @@ const UiSharedDepsNpm = require('@kbn/ui-shared-deps-npm');
 const { distDir: UiSharedDepsSrcDistDir } = require('./src/definitions');
 
 const MOMENT_SRC = require.resolve('moment/min/moment-with-locales.js');
-
-const REPO_ROOT = Path.resolve(__dirname, '..', '..', '..', '..', '..');
+const REPO_ROOT = require('child_process')
```

Comment:
I haven'r dig long enough but I think the cause for the bundle booms is probably related with this. I think we should try to re-use the previous setup or maybe `const { REPO_ROOT } = require('@kbn/repo-info');` . A given polyfill is being copied to the bundle instead of being referred and path mismatching is likely the cause

Reply:
I'll check, but this is a webpack config, it shouldn't end up in the bundles, no?